applications/vanilla/models/class.categorymodel.php

```diff
@@ -2045,12 +2045,7 @@ private function getOne($id) {
      * Get list of categories (disregarding user permission for admins).
      *
      * @since 2.0.0
-     * @access public
-     *
-     * @param string $OrderFields Ignored.
-     * @param string $OrderDirection Ignored.
-     * @param int $Limit Ignored.
-     * @param int $Offset Ignored.
+     * 
```

Comment:
```suggestion
     *
```

Coding standards complaint